egg/core/callbacks.py

```diff
@@ -69,6 +79,24 @@ def on_test_end(self, loss: float, logs: Interaction, epoch: int):
     def on_epoch_end(self, loss: float, logs: Interaction, epoch: int):
         if self.print_train_loss:
             self.aggregate_print(loss, logs, "train", epoch)
+        self.batch_info['cur_epoch'] += 1
+        self.batch_info['cur_batch'] = 0
+        print()
+
+    def on_batch(self, logs: Interaction, loss: float, data_len: int):
+
+        bi = self.batch_info
```

Comment:
reason to have bi saved here? Plus I think we would a better name for the variable